src/stores/models/Oracle.js

```diff
@@ -0,0 +1,90 @@
+import _ from 'lodash';
+import { defineMessages } from 'react-intl';
+import { OracleStatus, TransactionType, TransactionStatus, Phases } from '../../constants';
+import { satoshiToDecimal } from '../../helpers/utility';
+const { Pending } = TransactionStatus;
+const { bet, vote, setResult, finalize, withdraw } = Phases;
+
+const messages = defineMessages({
+  placeBet: { id: 'bottomButtonText.placeBet', defaultMessage: 'Place Bet' },
+  setResult: { id: 'str.setResult', defaultMessage: 'Set Result' },
+  arbitrate: { id: 'bottomButtonText.arbitrate', defaultMessage: 'Arbitrate' },
+  finalizeResult: { id: 'str.finalizeResult', defaultMessage: 'Finalize Result' },
```

Comment:
need to define message of `pending`. let me merge my PR first then can rebase and add here.